api3-whitepaper.tex

```diff
@@ -612,103 +612,66 @@ \subsection{API3 DAO}
 For the DAO to be able to assess its performance, the team must report to the DAO in great detail.
 These reports will have the additional benefit of providing accountability and allow the dAPI users and the general public to be able to audit the operations of API3 at all times.
 
-\subsection{dAPI monetization and API provider compensation}
-\label{sec:dapi-monetization-and-api-provider-compensation}
-
-API subscription fees are commonly paid monthly or annually, as this scheme suits both API providers and their clients.
-API3 will aim to follow the same scheme for dAPIs.
-To gain access to a dAPI, the user will pay a recurring subscription fee, which may either be fixed or customized for the user based on the specific use case.
-These prices will be determined by the respective team, and will include the premium if the user wants to receive the insurance service described in Section~\ref{sec:quantifiable-security-through-insurance}.
-The payment will be able to be made in any cryptocurrency, which will be received by the DAO in API3 tokens through a liquidity pool-based decentralized exchange.
-
-API providers will be compensated periodically at fixed rates, which will fit their existing pricing models.
-This will be done using stablecoins wherever possible, yet as mentioned in Section~\ref{sec:barriers-to-api-providers-operating-oracles}, some API providers categorically reject handling cryptocurrency as payment.
-In such cases, the DAO will provide a grant that will be paid out in return of the proof that the API provider is compensated in fiat by the grantee.
```

Comment:
This is mostly about dAPI revenue, which is covered in the staking section. We can revive the content (especially the part about API provider compensation) as it gets fleshed out.